cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -74,9 +73,11 @@ def _install(self):
         yum_install(sources['psycopg2_rpm_url'])
 
         files.copy_notice(POSTGRESQL_CLIENT)
-
-        self._create_postgres_group()
-        self._create_postgres_user()
+        db_server_username = \
+            config[POSTGRESQL_CLIENT].get('db_server_username', None)
+        if db_server_username == 'postgres' or not db_server_username:
```

Comment:
why not just "not db_server_username" ?

Reply:
In case it's left out empty, so it can be `postgres` by default